src/ServicePulse.Host/app/js/views/archive/controller.js

```diff
@@ -72,35 +73,53 @@
             vm.loadingData = false;
         };
 
+        var saveGroupOption = (amount, unit) => {
+            $cookies.put('archive_amount', amount);
+            $cookies.put('archive_unit', unit);
+        };
+
+        var getDefaultGroup = () => {
```

Comment:
Maybe `getSelectedArchiveGroup` or something to at least indicate what the group is.